### ISSUE TITLE
false indentation error raised after keyword feature

### ISSUE THREAD

**Issue by friedrith** [vsiakka/gherkin-lint#201]
Hi,
First of all, thank you for this project. I find gherkin very useful and a linter is always a good safeguard.

When using keyword feature, we are supposed to be able to indent like following example described in the [reference documentation](https://cucumber.io/docs/gherkin/reference/) :
```gherkin
Feature: Guess the word

  # The first example has two steps
  Scenario: Maker starts a game
    When the Maker starts a game
    Then the Maker waits for a Breaker to join

  # The second example has three steps
  Scenario: Breaker joins a game
    Given the Maker has started a game with the word "silky"
    When the Breaker joins the Maker's game
    Then the Breaker must guess a word with 5 characters
```

Scenarios may have 2-spaces indentation.

But using gherkin-lint version 3.3.6 (master version while writing this issue), I get an error saying that the indentation is wrong. The linter only accepts this indentation 

```gherkin
Feature: Guess the word

# The first example has two steps
Scenario: Maker starts a game
  When the Maker starts a game
  Then the Maker waits for a Breaker to join

 # The second example has three steps
 Scenario: Breaker joins a game
   Given the Maker has started a game with the word "silky"
   When the Breaker joins the Maker's game
   Then the Breaker must guess a word with 5 characters
```

Example to reproduce the bug: https://github.com/friedrith/gherkin-lint/tree/issue-indent

Is it something expected or a bug?

**Comment by vsiakka** [comment]
Hey @friedrith, the default indentation configuration for Scenarios is configured to `0` so this is expected. You can see the default indentation here: https://github.com/vsiakka/gherkin-lint#indentation
